spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "create" do
+    let(:user) {User.create(:first_name => "Rebecca", :last_name => "Holzschuh", :email => "rebeccaholzschuh@gmail.com", :username=> "rebecca")}
+    it "adds one user to the database" do
+      expect {user}.to change(User, :count).by(1)
+    end
+
+    it "adds the user to the database" do
+      User.create(:first_name => "Rebecca")
+      expect(User.last.first_name).to eq("Rebecca")
+    end
+
+    it "redirects back to the home page after saving a user" do
+      post :create, user: {:first_name => "Rebecca", :last_name => "Holzschuh"}
+      expect(response).to redirect_to root_path
+    end
+  end
+
+  before(:each) do
+    @user = User.create(:first_name => "Rebecca", :last_name => "Holzschuh", :email => "rebeccaholzschuh@gmail.com", :username=> "rebecca")
+  end
+
+  describe "show" do
+    subject {get :show}
+    it "renders show template" do
```

Comment:
We like our tests to assert something interesting, and not test the rails internals.  Things like it "renders the page without error" OR it "shows rebecca" would be much more useful.